base/ESMFL_Mod.F90

```diff
@@ -301,7 +301,7 @@ end subroutine ESMFL_StateGetField
 
 !BOP
 
-! !IROUTINE: ESMFL_GridCoordGet - retrieves the coordinates of a particular axis 
+! !IROUTINE: ESMFL_GridCoordGet - retrieves the coordinates of a particular axis in radians
```

Comment:
@rmontuoro We updated this doc string as well. We couldn't do it through the GitHub "make a suggestion" feature since it wasn't within three lines of a change...I guess?